endorse/endorse.go

```diff
@@ -198,9 +208,9 @@ func outputTdxMeasurement(_ *Context, tdx *epb.VMTdx) {
 	}
 }
 
-// Ovmf calculates the golden measurement of the given OVMF image, signs a document with the
+// Endorse calculates the golden measurement of the given OVMF and (if supplied) SVSM image, signs a document with the
 // measurement and associated metadata, submits it, and performs finalization.
-func Ovmf(ctx context.Context) error {
+func Endorse(ctx context.Context) error {
```

Comment:
endorse.Endorse is repetitive and generally against Go style. I would recommend, say, VirtualFirmware